mlflow/types/utils.py

```diff
@@ -238,7 +237,7 @@ def __call__(self, x):
             return DataType.string
         else:
             raise MlflowException(
-                "Unable to map 'np.object' type to MLflow DataType. np.object can"
+                "Unable to map 'object' type to MLflow DataType. object can"
```

Comment:
This might sound strange since everything is an object in Python.

Reply:
I think it sounds ok and is similar to how this sort of Exception message would be raised in other libraries.